ui/app/routes/evaluations/index.js

```diff
@@ -32,23 +32,56 @@ export default class EvaluationsIndexRoute extends Route {
     nextToken,
     pageSize,
     searchTerm,
-    // eslint-disable-next-line no-unused-vars
     status,
-    // eslint-disable-next-line no-unused-vars
     triggeredBy,
     type,
     qpNamespace: namespace,
   }) {
-    const generateFilter = () => {
+    const generateFilterExpression = () => {
       const searchFilter = searchTerm
         ? `ID contains "${searchTerm}" or JobID contains "${searchTerm}" or NodeID contains "${searchTerm}" or TriggeredBy contains "${searchTerm}"`
         : null;
       const typeFilter =
         type === 'client' ? `NodeID is not empty` : `NodeID is empty`;
+      const triggeredByFilter = `TriggeredBy contains "${triggeredBy}"`;
+      const statusFilter = `Status contains "${status}"`;
```

Comment:
This looks neat; we're generating plain-language filter expressions for our API? Could you provide some context into how this is being used?

Reply:
Not sure if this is what you are asking for, but here are the docs on the API filter expression syntax: https://www.nomadproject.io/api-docs#filtering

Reply:
Is this what you were hoping for @philrenaud?

I like following the Seeker/Stumbler paradigm and linking directly to the documentation and resources in code to help seekers.